src/Aspire.Hosting.AWS/SDKResourceExtensions.cs

```diff
@@ -46,6 +46,24 @@ public static IAWSSDKConfig WithRegion(this IAWSSDKConfig config, RegionEndpoint
         return config;
     }
 
+    /// <summary>
+    /// Disable validation for profile in sdk.
+    /// </summary>
+    /// <param name="config">An <see cref="IAWSSDKConfig"/> instance.</param>
+    public static IAWSSDKConfig WithNoValidation(this IAWSSDKConfig config)
```

Comment:
I think we should make this `WithValidation` and then take in a `enabled` parameter. Rather than having WithNoValidation, make it the inverse - i think it will be easier to read.

Then the user can call `WithValidation(false)`

Reply:
I think `WithSDKValidation` is better. who knows what validation will come to this in the future :)